modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderIT.java

```diff
@@ -129,6 +138,104 @@ public void testGeoIpDatabasesDownload() throws Exception {
         }
     }
 
+    public void testUseGeoIpProcessorWithDownloadedDBs() throws Exception {
+        // setup:
+        BytesReference bytes;
+        try (XContentBuilder builder = JsonXContent.contentBuilder()) {
+            builder.startObject();
+            {
+                builder.startArray("processors");
+                {
+                    builder.startObject();
+                    {
+                        builder.startObject("geoip");
+                        {
+                            builder.field("field", "ip");
+                            builder.field("target_field", "ip-city");
+                            builder.field("database_file", "GeoLite2-City.mmdb");
+                        }
+                        builder.endObject();
+                    }
+                    builder.endObject();
+                    // TODO: change geoip fixture to not return city db content for country and asn databases:
```

Comment:
I'd rather not have this big `TODOs` here, I can fix geoip fixture if you want before we merge this PR.
I wonder how stable/fragile this will be considering third-party test (where we hit real service and data may change eventually)

Reply:
> I can fix geoip fixture if you want before we merge this PR.

That would be great then also the other databases can be tested here instead just the city db.

> I wonder how stable/fragile this will be considering third-party test (where we hit real service and data may change eventually)

Not sure how to enforce that the test fixture stays inline with the actual api that infra is building.
Best thing I can come up with is that we share a spec, like we do with our rest apis. If any of us
make a change then we at least now about it.